index.js

```diff
@@ -474,6 +474,27 @@ class Squeakquel extends Datastore {
             findParams.attributes = includedFields.map((field) => {
                 let col = Sequelize.col(field);
 
+                // Temporary treatment to show correct trusted value.
+                // This subQuery is used on fiels of SELECT clause.
+                // This needs to delete after the trusted table generated.
+                if (field === 'trusted') {
+                    // TODO: config.table -> tableName
```

Comment:
After this [PR](https://github.com/screwdriver-cd/datastore-sequelize/pull/62) merged, this todo will fix.

Reply:
fiexd.